components/console/helpers/questionhelper.rst

```diff
@@ -289,26 +289,15 @@ from the command line, you need to set the helper input stream::
         // ...
         $commandTester = new CommandTester($command);
 
-        $helper = $command->getHelper('question');
-        $helper->setInputStream($this->getInputStream("Test\n"));
+        $commandTester->setInputs(array('Test'));
         // Equals to a user inputting "Test" and hitting ENTER
-        // If you need to enter a confirmation, "yes\n" will work
+        // If you need to enter a confirmation, adding a second input saying "yes" will work
 
         $commandTester->execute(array('command' => $command->getName()));
 
         // $this->assertRegExp('/.../', $commandTester->getDisplay());
     }
 
-    protected function getInputStream($input)
-    {
-        $stream = fopen('php://memory', 'r+', false);
-        fputs($stream, $input);
-        rewind($stream);
-
-        return $stream;
-    }
-
-By setting the input stream of the ``QuestionHelper``, you imitate what the
-console would do internally with all user input through the CLI. This way
-you can test any user interaction (even complex ones) by passing an appropriate
-input stream.
+By calling :method:`CommandTester::setInputs`, you imitate what the console would
+do internally with all user input through the CLI. This way you can test any
+user interaction (even complex ones) by passing the appropriate inputs.
```

Comment:
@xabbuh right, the behavior stays the same, we create a string from the given array of inputs,  (each input is separated via a break-line), then the `QuestionHelper` read the stream when `ask()` is called (via  `fgets`).
I added [a short explanation](https://github.com/symfony/symfony-docs/pull/6623/commits/51eb9d0cf544bc50301cf37c1c50abf09a4e1816), I hope it is clear enough.